Sources/TuistGenerator/GraphViz/GraphToGraphVizMapper.swift

```diff
@@ -147,15 +154,7 @@ extension GraphDependency {
 }
 
 extension GraphTarget {
-    fileprivate func isExternal(root: AbsolutePath) -> Bool {
-        isPathToTuistDependencyDirectory(path, root: root)
+    fileprivate func isExternal(root _: AbsolutePath) -> Bool {
+        project.isExternal
     }
```

Comment:
Can't we get rid of this helper and just do `project.isExternal`?

Reply:
Sure